docs/reference/7-docker/readme.mdx

```diff
@@ -0,0 +1,806 @@
+    docker compose -f ./Compose/pgsql/compose-sandbox-env.yml --env-file ./.env up -d
+    ```
+
+    **Powershell script**
+
+    ```powershell
+    ./sandbox-env-up.ps1
+    ```
+  </TabItem>
+  <TabItem value="sqlserver" label="SQL Server">
+    **Docker command**
+
+    ```powershell
+    docker compose -f ./Compose/mssql/compose-sandbox-env.yml --env-file ./.env up -d
+    ```
+
+    **Powershell script**
+
+    ```powershell
+    ./sandbox-env-up.ps1 -Engine SQLServer
+    ```
+  </TabItem>
+</Tabs>
+
+##### SingleTenant Environment
+
+<Tabs>
+  <TabItem value="postgresql" label="PostgreSQL" default>
+    **Docker command**
+
+    ```powershell
+    docker compose -f ./Compose/pgsql/SingleTenant/compose-single-tenant-env.yml --env-file ./.env up -d
+    ```
+
+    **Powershell script**
+
+    ```powershell
+    ./single-tenant-env-up.ps1
+    ```
+  </TabItem>
+  <TabItem value="sqlserver" label="SQL Server">
+    **Docker command**
+
+    ```powershell
+    docker compose -f ./Compose/mssql/SingleTenant/compose-single-tenant-env.yml --env-file ./.env up -d
+    ```
+
+    **Powershell script**
+
+    ```powershell
+    ./single-tenant-env-up.ps1 -Engine SQLServer
+    ```
+  </TabItem>
+</Tabs>
+
+##### SingleTenant with ODS Context Routes Environments
+
+<details>
+  <summary>PostgreSQL</summary>
+
+<Tabs>
+  <TabItem value="docker-command" label="Docker Command" default>
+    ```powershell
+    docker compose -f ./Compose/pgsql/SingleTenant-OdsContext/compose-single-tenant-odscontext-env.yml --env-file ./.env up -d
+    ```
+  </TabItem>
+  <TabItem value="powershell-script" label="Powershell Script">
+    ```powershell
+    ./single-tenant-odscontexts-up.ps1
+    ```
+  </TabItem>
+</Tabs>
+
+</details>
+
+Notice that the compose file provided in the repository is an example that
+defines ODS for the years 2022 and 2023.  You can generate the compose file for
+the desired school years.
+
+<details>
+  <summary>Generating Compose File</summary>
+
+To customize for desired school years, you can use the Mustache template
+provided in the repository.
+
+Navigate to the folder where you cloned/downloaded the Ed-Fi-ODS-Docker
+repository and go to the sub-folder
+_Compose-Generator/Alpine/templates/pgsql/SingleTenant-OdsContext_; in there,
+you will find example configuration files that instruct the Mustache template on
+how to generate the SingleTenant with OdsContexts Docker Compose file.
+
+Copy `parameters.example.yml` and update it as you see fit; the available
+options are:
+
+* **odsContextRouteTemplate** - Defines the route parameter template used to
+    specify the school year.
+
+    {/* Configure it following [route
+    constraints](<https://techdocs.ed-fi.org/docker> run --rm -v
+    c:%5CEd-Fi-ODS-Docker%5CCompose-Generator%5CAlpine%5Ctemplates%5Cpgsql%5Cparameters-year-specific.yml:/parameters
+    -v C:%5CEd-Fi-ODS-Docker%5CCompose%5Cpgsql%5CSingleTenant-OdsContext:/output
+    edfialliance/ods-compose-generator). */}
```

Comment:
[Lines 421-425]

I'm not exactly sure what this link is supposed to do. Un-commented, it causes a build error.
There is another link like this further down.

Reply:
Someone had accidentally pasted a powershell command as a link. I think I found the intended URL.